Gemfile

```diff
@@ -3,14 +3,3 @@ source 'https://rubygems.org'
 
 # Specify your gem's dependencies in krane.gemspec
 gemspec
-
-gem 'pry'
-gem 'pry-byebug'
-gem 'rubocop'
-gem 'timecop'
-gem 'byebug'
-gem 'codecov', require: false
```

Comment:
Can/should we drop `require 'X'`s  in test_helper.rb

Reply:
Good question. I don't have a single authoritative source to point you to, but reading various things on the internet indicates that:
- Nothing from the gemspec is required by default.
- Even things in the gemfile aren't "magically" required unless you do `Bundler.require` somewhere, which we don't. Since our test helper does the `require`s individually anyway, our old `require: false` wasn't doing anything for us.